5.5/examples/replica/mysql_replica.json

```diff
@@ -158,11 +158,11 @@
                 "env": [
                   {
                     "name": "MYSQL_MASTER_USER",
```

Comment:
i don't think it's worth changing it, but i'm also not clear why even the parameter name really needed to be changed.